modes/off_mode/off_page.py

```diff
@@ -1,8 +1,10 @@
 from tkinter import Frame
 from modes.page import Page
 from models.model import Model
+from modes.efficiency_mode.home import Home
 
 
-class Off(Page):
+class Off(Home):
```

Comment:
if home is specific to the efficiency mode then should all pages really inherit it? probs not

Reply:
this is just a placeholder for now